lib/Drush/Config/StorageWrapper.php

```diff
@@ -0,0 +1,259 @@
+<?php
+
+/**
+ * @file
+ * Definition of Drush\Config\StorageWrapper.
+ */
+
+namespace Drush\Config;
+
+use Drupal\Core\Config\StorageInterface;
+
+class StorageWrapper implements StorageInterface {
+
+  /**
+   * The storage container that we are wrapping.
+   *
+   * @var \Drupal\Core\Config\StorageInterface
+   */
+  protected $storage;
+  protected $filters;
+
+  /**
+   * Create a StorageWrapper with some storage and a filter.
+   */
+  function __construct($storage, $filterOrFilters) {
+    $this->storage = $storage;
+    $this->filters = is_array($filterOrFilters) ? $filterOrFilters : array($filterOrFilters);
+  }
+
+  /**
+   * Returns whether a configuration object exists.
+   *
+   * @param string $name
+   *   The name of a configuration object to test.
+   *
+   * @return bool
+   *   TRUE if the configuration object exists, FALSE otherwise.
+   */
+  public function exists($name) {
```

Comment:
all those phpdocs for the public methods from the interface could be just {@inheritdoc} ;-)